README.md

```diff
@@ -35,130 +31,70 @@ Welcome to the Netlify CLI! The new 2.0 version (now in beta) was rebuilt from t
 </details>
 <!-- AUTO-GENERATED-CONTENT:END -->
 
-## Installation
```

Comment:
We can't get rid of installation and a basic usage example from the README.  Its slightly duplicate content, but provides expected context if you arrive to the CLI on the GitHub repo or npm page.

Reply:
I agree with bret. We cannot remove this

Reply:
👍 Updating now